.storybook/themes/commerce.scss

```diff
@@ -378,6 +378,12 @@
 					font-size: var(--body-font-size-small),
 					line-height: var(--body-line-height-small),
 				),
+				divider: (
+					margin-top: var(--global-spacing-2),
```

Comment:
I assume that there should be both top and bottom margin?  Otherwise, doesn't the border sit on the bottom?

Reply:
Fixed it in [5a57118](https://github.com/WPMedia/arc-themes-components/pull/145/commits/5a571189a41ea73014f6fa5f5bbe190a940e7afa)

Reply:
I might be missing context on this (Nick's out). But it doesn't look like there's any margin on the component in the designs. Let me know if adding margin to the base theme for divider was a requirement I'm missing  


<img width="467" alt="Screen Shot 2022-10-07 at 12 59 10" src="https://user-images.githubusercontent.com/5950956/194620692-cf3c3650-e836-4fdf-bd0d-ceca16456e82.png">

Reply:
https://github.com/WPMedia/arc-themes-components/pull/145#discussion_r991261273